docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -0,0 +1,68 @@
+# Humsci Basic (humsci_basic) Theme
+
+Humsci Basic (humsci_basic) is a theme that can be used as a base for all future H&S themes. We have pulled patterns and templates for Humsci Basic from Stanford Basic (stanford_basic) and Stanford University Humsci Theme (su_humsci_theme) Themes. Humsci Basic integrates selectively with Decanter Sass, styles, and patterns.
+
+## Requirements
+
+- Drupal 8.7.10
+- Node 10.15+
+- Decanter 5.1.0, soon to be upgraded to 6.0.0
```

Comment:
6 has been released https://github.com/SU-SWS/decanter/releases/tag/6.0.0

Reply:
Updating to say 6, since we made that change in a recent PR.